user/languages/_includes/modern-c-cpp-toolchain.md

```diff
@@ -0,0 +1,130 @@
+### Modern C/C++ and Toolchain Versioning
+
+If you need a more modern toolchain or more granular control over your tools, you will have to handle that yourself. Fortunately, the community has already whitelisted the [repositories](https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json) and [packages](https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise) necessary for most such upgrades.
```

Comment:
> Fortunately, the community has already whitelisted…

I don't think this works here. Who listed the packages is probably not important to the reader, but just the fact that the sources and packages are available for use. (Who is "the community"?)

The instructions to request new sources/packages are given when you run jobs on containers, but I now see that the documentation https://docs.travis-ci.com/user/installing-dependencies does not contain that. We should fix it.